lib/rucio/tests/temp_factories.py

```diff
@@ -116,6 +119,27 @@ def make_mock_rse(self):
     def make_xroot_rse(self):
         return self._make_rse(scheme='root', protocol_impl='rucio.rse.protocols.xrootd.Default')
 
+    def fetch_containerized_rse(self, rse):
```

Comment:
I don't think this function is t a correct place in this class. It doesn't use any functions provided by the class (notably: cleanup of created RSEs). And the functionality it brings is orthogonal to the existing function of the class. I'd vote to move the function as a stand-alone fixture "containerized_rses" (or similar naming).